Gemfile

```diff
@@ -36,7 +36,9 @@ group :test do
   gem "capybara"
   gem "database_cleaner"
   gem "formulaic"
+  gem "jet_black"
   gem "launchy"
+  gem "sack_race", github: "nickcharlton/sack_race"
```

Comment:
Before merging, I'll cut the full version of v1.0.0. It's at v1.0.0.rc1 (because it's been working well here so far).